synapse/tests/test_tools_dumprows_loadrows.py

```diff
@@ -0,0 +1,316 @@
+# -*- coding: utf-8 -*-
+# XXX Update Docstring
+"""
+synapse - test_tools_dumprows_loadrows.py
+Created on 7/26/17.
+
+Unittests for the dumprows and loadrows tools.
+"""
+import synapse.axon as s_axon
+
+import synapse.tools.dumprows as s_dumprows
+import synapse.tools.loadrows as s_loadrows
+
+from synapse.tests.common import *
+
+log = logging.getLogger(__name__)
+
+class DumpRowsTest(SynTest):
+
+    def make_sql_genrows_json(self, fp):
+        d = {'slicebytes': 2, 'incvalu': 4}
+        with open(fp, 'wb') as f:
+            f.write(json.dumps(d, indent=2, sort_keys=True).encode())
+
+    def test_simple_use(self):
+        outp = self.getTestOutp()
+        with self.getTestDir() as temp:
+            fp = os.path.join(temp, 'dumpfile.msgpk')
+            new_db = os.path.join(temp, 'test.db')
+            sqlite_url = 'sqlite:///{}'.format(new_db)
+            with s_cortex.openurl(sqlite_url) as core:
+                self.true(core.isnew)
+                core.setBlobValu('syn:test:tel', 8675309)
+                core.formTufoByProp('inet:ipv4', 0x01020304)
+            # Now dump that sqlite core
+            argv = ['-s', sqlite_url, '-o', fp]
+            ret = s_dumprows.main(argv, outp)
+            self.eq(ret, 0)
+
+            # Now ensure our .msgpk file is correct
+            with open(fp, 'rb') as fd:
+                gen = msgpackfd(fd)
+                evt = next(gen)
+                self.eq(evt[0], s_dumprows.ROWDUMP)
+                self.eq(evt[1].get('rows:compress'), False)
+                self.eq(evt[1].get('synapse:rows:output'), fp)
+                self.eq(evt[1].get('synapse:cortex:input'), sqlite_url)
+                self.eq(evt[1].get('synapse:cortex:blob_store'), False)
+                self.eq(evt[1].get('synapse:cortex:revstore'), False)
+                self.eq(evt[1].get('python:version'), s_compat.version)
+                self.isin('synapse:version', evt[1])
+                evt = next(gen)
+                self.eq(evt[0], s_dumprows.ADDROWS)
+                self.isin('rows', evt[1])
+                rows = evt[1].get('rows')
+                self.isinstance(rows, tuple)
+                self.isinstance(rows[0], tuple)
+                self.eq(len(rows[0]), 4)
+                # Expensive but worth checking
+                event_types = set()
+                event_types.add(evt[0])
+                total_rows = 0
+                for evt in gen:
+                    event_types.add(evt[0])
+                    if 'rows' in evt[1]:
+                        total_rows = total_rows + len(evt[1].get('rows'))
+                self.gt(total_rows, 1000)
+                self.eq(event_types, {s_dumprows.ADDROWS})
+
+    def test_simple_compress(self):
+            outp = self.getTestOutp()
+            with self.getTestDir() as temp:
+                fp = os.path.join(temp, 'dumpfile.msgpk')
```

Comment:
i've been using .mpk files so far as the convention name.  any reason not to normalize this one too?